test/workspaces/copy_files/jbuild

```diff
@@ -0,0 +1,8 @@
+(jbuild_version 1)
+
+(ocamllex (lexer1))
```

Comment:
It seems nicer to me if this stanza is next to the .mll. I think we should move it lexers/jbuild and change the glob to `lexers/*.{ml,mli}`